### PR TITLE
Update descriptions on what special characters are allowed for DB pas…

### PR DESCRIPTION
**Pull request by errcode1202**

### CODE REVIEW

File: templates/quickstart-bitbucket-dc-with-vpc.template.yaml [hunk 1]
```diff
@@ -404,8 +404,8 @@ Parameters:
     Type: String
   DBPassword:
     AllowedPattern: '(?=^.{6,255}$)((?=.*\\d)(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[^A-Za-z0-9])(?=.*[a-z])|(?=.*[^A-Za-z0-9])(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[A-Z])(?=.*[^A-Za-z0-9]))^.*'
-    ConstraintDescription: Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
-    Description: Password for the Bitbucket user ('atlbitbucket') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
+    ConstraintDescription: 'Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, and 1 of the following symbols: ! # $ { * : [ = , ] - _ @ + % &'
```
Comment (errcode1202) [inline templates/quickstart-bitbucket-dc-with-vpc.template.yaml:407]: This wording seemed wrong from the start, in that it was referring to the BitBucket user `atlbitbucket` and didnt seem to be related to the DB password. I've updated it as can be seen but let me know if this is not right.

File: templates/quickstart-bitbucket-dc-with-vpc.template.yaml [hunk 2]
```diff
@@ -388,8 +388,8 @@ Parameters:
     AllowedPattern: >-
       ^(?=^.{8,255}$)(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^A-Za-z0-9])(?!.*[@/"']).*$
     ConstraintDescription: >-
-      Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
-    Description: Password for the master ('postgres') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
+      Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, and 1 of the following symbols: ! # $ { * : [ = , ] - _ @ + % &
```
Comment (nanux) [inline templates/quickstart-bitbucket-dc-with-vpc.template.yaml:391]: Do you know if there was any issue with the `@` symbol? I wonder why it was excluded previously.

File: templates/quickstart-bitbucket-dc.template.yaml [hunk 3]
```diff
@@ -378,8 +378,8 @@ Parameters:
     Type: String
   DBPassword:
     AllowedPattern: '(?=^.{6,255}$)((?=.*\\d)(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[^A-Za-z0-9])(?=.*[a-z])|(?=.*[^A-Za-z0-9])(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[A-Z])(?=.*[^A-Za-z0-9]))^.*'
-    ConstraintDescription: Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
-    Description: Password for the Bitbucket user ('atlbitbucket') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol
+    ConstraintDescription: 'Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, and 1 of the following symbols: ! # $ { * : [ = , ] - _ @ + % &'
```
Comment (errcode1202) [inline templates/quickstart-bitbucket-dc.template.yaml:381]: Same as previous comment - This wording seemed wrong from the start, in that it was referring to the BitBucket user `atlbitbucket` and didnt seem to be related to the DB password. I've updated it as can be seen but let me know if this is not right.

File: templates/quickstart-bitbucket-dc-with-vpc.template.yaml (hunk 1 above)
Comment (nanux) [reply templates/quickstart-bitbucket-dc-with-vpc.template.yaml:407]: This is a leftover from previous setup when BB included `DB` resource in the main template. With nested templates, we've aligned the DB usernames. Good catch.

**Review by nanux — APPROVED** [review]

File: templates/quickstart-bitbucket-dc-with-vpc.template.yaml (hunk 2 above)
Comment (errcode1202) [reply templates/quickstart-bitbucket-dc-with-vpc.template.yaml:391]: So based on the taskcat `template_params.py` file `@` should be supported as defined by the variable: `specialchars = "!#$&{*:[=,]-_%@+"`. No sure why it was missed previously.

**Pull request merged**